docs/data-sources/azure_cloud_service_deployment_targets.md

```diff
@@ -45,44 +45,44 @@ data "octopusdeploy_azure_cloud_service_deployment_targets" "example" {
 
 ### Read-Only
 
-- `azure_cloud_service_deployment_targets` (Block List) A list of Azure cloud service deployment targets that match the filter(s). (see [below for nested schema](#nestedblock--azure_cloud_service_deployment_targets))
+- `azure_cloud_service_deployment_targets` (List of Object) A list of Azure cloud service deployment targets that match the filter(s). (see [below for nested schema](#nestedatt--azure_cloud_service_deployment_targets))
 - `id` (String) An auto-generated identifier that includes the timestamp when this data source was last modified.
 
-<a id="nestedblock--azure_cloud_service_deployment_targets"></a>
+<a id="nestedatt--azure_cloud_service_deployment_targets"></a>
 ### Nested Schema for `azure_cloud_service_deployment_targets`
 
 Read-Only:
 
 - `account_id` (String)
 - `cloud_service_name` (String)
 - `default_worker_pool_id` (String)
-- `endpoint` (List of Object) (see [below for nested schema](#nestedatt--azure_cloud_service_deployment_targets--endpoint))
```

Comment:
These changes are a result of the Optional -> Computed changes in [this PR](https://github.com/OctopusDeployLabs/terraform-provider-octopusdeploy/commit/3a8a027a764cda034b8840bbba6df254e5f0d6b4).